src/pg_to_evalscript/javascript_processes/add_dimension.js

```diff
@@ -1,6 +1,294 @@
+function _add_dimension(arguments) {
+  const { data } = arguments;
+  const dataCube = new DataCube(data, "bands_name", "temporal_name");
```

Comment:
If I understand correctly, this and the entire `DataCube` source code was added for testing purposes. That's not how it should be done...

I mean the `DataCube` code is now multiplied in many places, if it changes it won't change here, it's loaded in the generated evalscript already so it's not needed in every process etc etc.

How it should be done:

As one can see, the `run_process` function accepts the source code of a process and adds a line that calls the method. The method that runs processes that are (partially) defined on the Datacube should:

1. Loads DataCube source code from `DataCube.js`
2. Loads the process definition
3. Adds a line which generates a testing datacube
4. Calls the function with the generated datacube in the argument

```python
{loaded_datacube_code}
{loaded_process_code}
const d = new DataCube({example_input}, ...)
stdout(
    {process_id}(dict("data", d))
)
```
^ used `dict` instead of curly braces for the dictionary so it's clear it isn't a parameter.

If you are not sure what's the purpose of some code or what this library does as a whole, feel free to ask.

Reply:
Should be fixed now